### ISSUE TITLE
Не все варианты начальной формы слова

### ISSUE THREAD

**Issue by Derinhelm** [kmike/pymorphy2#124]
Добрый вечер!
У morph.parse('содержат')
почему-то возвращается начальная форма содержимый, и больше вариантов не возвращается.
[Parse(word='содержат', tag=OpencorporaTag('VERB,impf,tran plur,3per,pres,indc'), normal_form='содержимый', score=1.0, methods_stack=((<DictionaryAnalyzer>, 'содержат', 3118, 33),))]
Подскажите пожалуйста, почему не учитывается вариант начальной формы - содержать(они содержат).

**Comment by opennota** [comment]
Потому что в словаре OpenCorpora нет?

https://github.com/OpenCorpora/opencorpora/issues

**Comment by buriy** [comment]
> Потому что в словаре OpenCorpora нет?
> 
> https://github.com/OpenCorpora/opencorpora/issues
Нет, в словаре всё нормально: http://opencorpora.org/dict.php?act=edit&id=328412
С 2012 года: http://opencorpora.org/dict_history.php?lemma_id=328412

**Comment by opennota** [comment]
```
<type id="3">INFN-VERB</type>
<type id="4">INFN-PRTF</type>
<link id="230398" from="328413" to="328412" type="3"/>
<link id="268120" from="388750" to="328413" type="4"/>
```

Кажется, в словаре OpenCorpora в последнем link перепутаны местами from и to. "Содержимый" должен превращаться в "содержать", а не наоборот.

Поменял, собрал словарь с помощью build-dict.py из pymorphy2-dicts, указал MorphAnalyzer'у path=, заработало:

содержать VERB,impf,tran plur,3per,pres,indc

**Comment by Derinhelm** [comment]
Спасибо большое!

**Comment by Shimorina** [comment]
Спасибо, я исправила в словаре OpenCorpora, теперь [связь](http://opencorpora.org/dict.php?act=edit&id=328413) в нужном направлении. Завтра новая версия словаря будет доступна на [сайте](http://opencorpora.org/?page=downloads).

**Comment by buriy** [comment]
@opennota может ли это быть массовым явлением? может, как-то можно проверить сразу все связи в базе на правильность направления?

**Comment by opennota** [comment]
Таких, чтобы по леммам выходило XXX-YYY, а в типе ссылок YYY-XXX, больше нет.
